conjure-java-undertow-runtime/src/main/java/com/palantir/conjure/java/undertow/runtime/ConjureHandler.java

```diff
@@ -88,11 +94,35 @@ public ConjureHandler delete(String template, HttpHandler handler) {
 
     private ConjureHandler register(HttpString method, String template, HttpHandler handler) {
         HttpHandler current = handler;
-        String endpoint = method + " " + template;
-        for (BiFunction<String, HttpHandler, HttpHandler> wrapper : WRAPPERS) {
+        EndpointDetails endpoint = new EndpointDetails(method, template);
+        for (BiFunction<EndpointDetails, HttpHandler, HttpHandler> wrapper : WRAPPERS) {
             current = wrapper.apply(endpoint, current);
         }
         routingHandler.add(method, template, current);
         return this;
     }
+
+    private static final class EndpointDetails {
+
+        private final HttpString method;
+        private final String template;
+
+        EndpointDetails(HttpString method, String template) {
+            this.method = method;
+            this.template = template;
+        }
+
+        HttpString getMethod() {
```

Comment:
IMO, since this is a private class it's fine to omit the getters and just access the fields directly